rise-financial-es6.js

```diff
@@ -108,14 +109,24 @@
       }, 60000 );
     }
 
+    _onDataPingReceived() {
+      this._dataPingReceived = true;
+
+      if ( this._goPending ) {
```

Comment:
@donnapep I don't think we should be calling `go()` in these "Received" handlers unless a `go()` call is pending by way of the consumer of the component has executed a `go()` call. That's why it's conditional, otherwise the component ends up calling `go()` multiple times unnecessarily which shouldn't happen.